README.md

```diff
@@ -193,15 +194,14 @@ If you are building on a network behind a proxy, set the environment variable
 set HTTP_PROXY=http://your-proxy-name:80
 ```
 
-Now you are ready to build the C# 4.0/4.5 version of Bond. Open the solution
-file `cs\cs.sln` in Visual Studio and build as usual. The C# unit tests can
+Now you are ready to build the C# version of Bond. Open the solution file
+`cs\cs.sln` in Visual Studio and build as usual. The C# unit tests can
 also be run from within the solution.
 
-To build the .NET Core version of Bond, run the build script. The `-Test`
-switch is used to run the unit tests as well.
+To build using the .NET Core SDK:
 
 ```bash
-.\cs\dnc\build -Test
+dotnet build cs.sln
```

Comment:
Should this be `cs\cs.sln` or `cd cs` before building?

Reply:
fixed